src/models/invites-ignorer.ts

```diff
@@ -144,10 +159,10 @@ export class IgnoredInvites {
      * @param roomId The room to which the user is invited.
      * @returns A rule matching the entity, if any was found, `null` otherwise.
      */
-    public async getRuleForInvite({ sender, roomId }: {
+    public getRuleForInvite({ sender, roomId }: {
```

Comment:
this is a breaking change because the class is not marked as unstable

Reply:
Well, this is a breaking change on my code that afaict isn't used anywhere yet, because the react-sdk patch was rejected because of `async` :)

Reply:
The issue is this wasn't marked experimental and there's 70+ dependants on matrix-js-sdk, any of which could use a release where the API was synchronous and then have major issues when updating within the same semver range, so it is a breaking change.

Reply:
@t3chguy So how do I handle this change? Should I add a new method instead of replacing the old one?

Reply:
Ideally yes, or mark it as X-Breaking-Change but then it might get held until we have a few to go together for a major version bump